Project.toml

```diff
@@ -28,8 +28,8 @@ ImageMagick = "1.1"
 JSServe = "1.2"
 Observables = "0.3"
 ShaderAbstractions = "0.2.1"
-StaticArrays = "0.12, 1.0"
-julia = "1.3"
+StaticArrays = "0.12, 1.0.2"
```

Comment:
Why did you specify the minor version in `1.0.2`? That will force compatibility only with that patch release from the `1.0.x` series (which I don't think it's a good idea).

Reply:
I've changed it again and it works.